app/src/express/express.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
```

Comment:
[OPTIONAL] I'd rather change folder structure of your project:
src/ - public client code
webpack/ - webpack configuration
server/ - code of your server
package.json